src/graph_coalescence/graph_coalescer.py

```diff
@@ -117,36 +115,43 @@ def coalesce_by_graph(opportunities, predicates_to_exclude=None, coalesce_thresh
                 threshold = len(nodes)
             if i >= threshold:
```

Comment:
Do we still want/need this?  I don't think so...

Reply:
I think we do unless the enrichment now includes precise genes, proteins/ entities of interest. If the remote Redis still cuts off the interesting entities, it's okay to increase the threshold.